release/ray_release/test_automation/ci_state_machine.py

```diff
@@ -74,6 +75,9 @@ def _create_github_issue(self) -> None:
         for failure in recent_failures:
             body += f"\t- {failure.url}\n"
         body += "\nManaged by OSS Test Policy"
+        body += "\n---"
```

Comment:
what does this `---` do magically? why it is required for showing up in go/flaky?

Reply:
totally